internal/reflect/options.go

```diff
@@ -18,3 +18,6 @@ type Options struct {
 	// returning errors. Numbers will always be rounded towards 0.
 	AllowRoundingNumbers bool
 }
+
+type OutOfOptions struct {
```

Comment:
Is `OutOfOptions` needed at all now?

Reply:
No, I left it in case, but it's all internal, anyways, so I guess there's no reason not to YAGNI it.